runtime/compiler/z/codegen/J9S390CHelperLinkage.hpp

```diff
@@ -122,6 +122,10 @@ class S390CHelperLinkage : public TR::Linkage
       return buildDirectDispatch(callNode, NULL, returnReg);
       }
 
-   TR::Register *buildDirectDispatch(TR::Node *callNode, TR::RegisterDependencyConditions** deps, TR::Register *returnReg=NULL);
+   TR::Register *buildDirectDispatch(TR::Node *callNode,
+                                     TR::RegisterDependencyConditions** deps,
+                                     TR::Register *returnReg=NULL,
+                                     bool forceFastPath = false,
+                                     TR_RuntimeHelper helperIndex = TR_S390numRuntimeHelpers);
```

Comment:
Could we move the documentation here such that it is with the definition of the API and document `foreFastPath` as well?

Reply:
Documentation is moved to the `hpp` file with `forceFastPath` added.